### ISSUE TITLE
Handle Status Code

### ISSUE THREAD

**Issue by engshorouq** [GSG-G7/read-the-movie#49]
https://github.com/GSG-G7/read-the-movie/blob/d4b8d85faef26896ec50b7bed40f4e11d9d385c6/js/xhr.js#L18-L20
the APIs send several status codes not just 200 and you must handle it.
you must handle if the response sends 200 and others status code like that at minimum.
```
if(xhr.readyState === 4){
            if(xhr.status === 200) {
                //do somthing
            } else {
                // do something
            }
        }
```

**Comment by Amoodaa** [comment]
yes thanks and we'll work on it, sorry! i missed the day2 workshop
